addons/website/security/ir.model.access.csv

```diff
@@ -1,6 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_website_public,website,website.model_website,,1,0,0,0
-access_website,website,website.model_website,group_website_designer,1,1,1,1
+access_website_designer,website,website.model_website,group_website_designer,1,1,1,1
```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/website/security/ir.model.access.csv#L10